notebooks/tutorials/1_getting_started_lesson3/notebook.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pipeline API"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Pipeline API is a powerful tool for cross-sectional analysis of asset data. It allows us to define a set of calculations on multiple data inputs and analyze a large amount of assets at a time. Some common uses for the Pipeline API include:  \n",
+    "\n",
+    "* Selecting assets based on filtering rules  \n",
+    "* Ranking assets based on a scoring function  \n",
+    "* Calculating portfolio allocations  \n",
```

Comment:
Is it correct to say that we use Pipeline to calculate portfolio allocations? Maybe in the case of a factor-weighted portfolio, but it seems to me that this is a follow-up step once you have your Pipeline output.